tools/slang-test/slang-test-main.cpp

```diff
@@ -4617,8 +4617,8 @@ SlangResult innerMain(int argc, char** argv)
 int main(int argc, char** argv)
 {
     const SlangResult res = innerMain(argc, argv);
-
     slang::shutdown();
+    Slang::RttiInfo::deallocateAll();
```

Comment:
We may need a document for how to properly shut everything down and deallocate cleanly.
Many user applications will want to shutdown properly to check the memory leaks of their own applications.

Reply:
This is only needed in slang-test-main because slang-test uses the RTTI functionality directly through static linking, so there is a different instance of slang::RTTIInfo that needs to be cleaned up separately. Normal users of slang.dll shouldn't need to do this.